_includes/payment-resource.md

```diff
@@ -107,7 +103,36 @@ Content-Type: application/json
             "method": "GET",
             "href": "{{ page.front_end_url }}/{{ api_resource }}/core/scripts/client.js?token={{ page.payment_token }}",
             "rel": "view-payment"
-        }{% else %},
+            "contentType:": "application/javascript"
+        },
+        {% when "trustly" %}
```

Comment:
Please ensure that this `when` doesn't introduce extra line-feeds. If it does, try adding `-` to the liquid tags to remove whitespace.

```suggestion
        {%- when "trustly" -%}
```